primitives/consensus/slots/Cargo.toml

```diff
@@ -16,11 +16,13 @@ targets = ["x86_64-unknown-linux-gnu"]
 codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = ["derive"] }
 sp-runtime = { version = "3.0.0", default-features = false, path = "../../runtime" }
 sp-arithmetic = { version = "3.0.0", default-features = false, path = "../../arithmetic" }
+serde = { version = "1.0.123", features = ["derive"], default-features = false }
```

Comment:
Needs to be optional and only enabled under `std`, same thing that was done in `primitives/consensus/babe`.